ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthPeer.java

```diff
@@ -210,17 +210,17 @@ public void removeChainEstimatedHeightListener(final long listenerId) {
   }
 
   public void recordRequestTimeout(final int requestCode) {
-    LOG.debug("Timed out while waiting for response from peer {}", this.getShortNodeId());
+    LOG.debug("Timed out while waiting for response from peer {}...", this.getShortNodeId());
     LOG.trace("Timed out while waiting for response from peer {}", this);
-    reputation.recordRequestTimeout(requestCode).ifPresent(this::disconnect);
+    reputation.recordRequestTimeout(requestCode);
   }
 
   public void recordUselessResponse(final String requestType) {
     LOG.debug(
-        "Received useless response for request type {} from peer {}",
+        "Received useless response for request type {} from peer {}...",
         requestType,
         this.getShortNodeId());
-    reputation.recordUselessResponse(System.currentTimeMillis()).ifPresent(this::disconnect);
+    reputation.recordUselessResponse(System.currentTimeMillis());
```

Comment:
this change is what's breaking the unit test, haven't figured out exactly why

Reply:
reverted this change - think it's better off as a separate PR. hunch is the test breaks because the block request task isn't switching to the new peer

Reply:
no... the secondPeer has no outstanding requests